pkg/proxy/proxy.go

```diff
@@ -145,61 +255,18 @@ func ConvertTestName(n string) string {
 	return tf
 }
 
-//DeleteLoadTestHandler deletes load test CR
-func DeleteLoadTestHandler(loadTestClient loadTestV1.LoadTestInterface) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		logger := mPkg.GetLogger(r.Context())
-
-		ctx, cancel := context.WithTimeout(context.Background(), loadtest.KubeTimeout)
-		defer cancel()
-
-		ltID := chi.URLParam(r, loadTestID)
-		logger.Debug("Deleting loadtest", zap.Any("ltID", ltID))
-
-		err := loadtest.DeleteLoadTestCR(ctx, loadTestClient, ltID, logger)
-		if err != nil {
-			logger.Error("Could not delete load test with error:", zap.Error(err))
-			render.Render(w, r, cHttp.ErrResponse(http.StatusBadRequest, err.Error()))
-			return
-		}
-
-		render.NoContent(w, r)
-	})
-}
-
-//GetLoadTestHandler returns the loadtest CR info.
-//For now it returns only created namespace,
-//will be extended when more business logic added
-func GetLoadTestHandler(loadTestClient loadTestV1.LoadTestInterface) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		logger := mPkg.GetLogger(r.Context())
-
-		ctx, cancel := context.WithTimeout(context.Background(), loadtest.KubeTimeout)
-		defer cancel()
-
-		ltID := chi.URLParam(r, loadTestID)
-		logger.Debug("Retrieving info for loadtest", zap.Any("ltID", ltID))
-
-		result, err := loadtest.GetLoadtestCR(ctx, loadTestClient, ltID, logger)
-		if err != nil {
-			logger.Error("Could not get load test info with error:", zap.Error(err))
-
-			if k8sErr, ok := err.(k8sAPIErrors.APIStatus); ok && k8sErr.Status().Code == http.StatusNotFound {
-				render.Render(w, r, cHttp.ErrResponse(http.StatusNotFound, err.Error()))
-				return
-			}
+func doRequest(req *restClient.Request) ([]byte, error) {
+	stream, err := req.Stream(context.Background())
+	if err != nil {
+		return []byte{}, fmt.Errorf("error in opening stream: %w", err)
```

Comment:
As I remember `nil` is valid return value for any slice type and in this case it is more efficent as you do not need to create ampty slice

```suggestion
		return nil, fmt.Errorf("error in opening stream: %w", err)
```